test/constexpr_core.cpp

```diff
@@ -179,8 +179,8 @@ RANGES_CXX14_CONSTEXPR auto test_non_member_f(Sequence1234&& a) -> bool {
 RANGES_CXX14_CONSTEXPR auto test_array() -> bool {
     test::array<int, 4> a{{1, 2, 3, 4}};
 
-    auto beg = ranges::begin(a);
-    auto three = ranges::next(beg, 2);
+    // auto beg = ranges::begin(a);
+    // auto three = ranges::next(beg, 2);
```

Comment:
These lines are doing something (the code will only compile if begin and next are constant expressions). By uncommenting these lines, this is no longer tested.

The fix here is not to comment out these lines, but to uncomment the ones below. 
I'll see if I can do this in the constexpr algorithms branch by using safe_int instead of int all over the place in constexpr test.